app/README.md

```diff
@@ -2,11 +2,14 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
```

Comment:
Let's also document all the choices we've made and the reasoning behind them in this file.